app/src/main/res/values-fa/strings.xml

```diff
@@ -0,0 +1,173 @@
+    <string name="pref_proot_delete_debug_file_title">پاک کردن پرونده دیباگ</string>
+
+    <!-- Help Activity -->
+    <string name="beta_message">با مشکلی مواجه شده یا درخواست افزودن ویژگی جدیدی دارید؟ صفحه گیت‌هاب ما بهترین مکان برای دریافت سریع پاسخ است.</string>
+    <string name="supported_services">خدماتی که در حال حاضر پشتیبانی می‌شوند:</string>
+    <string name="terminology">واژه‌شناسی:</string>
+    <string name="desc_apps_title">کاره‌ها:</string>
+    <string name="desc_apps_message">کاره‌ها، آسان‌ترین راه برای اجرای مرسوم‌ترین توزیع‌های گنو/لینوکسی هستند.</string>
+    <string name="desc_sessions_title">جلسات:</string>
+    <string name="desc_sessions_message">جلسات، کارسازی را بر روی فایل‌سیستم شما اجرا کرده و به شما اجزاه می‌دهند با کمک برنامه‌های کارخواه دیگر، به آن متصل شوید.</string>
+    <string name="desc_filesystems_title">فایل‌سیستم‌ها:</string>
+    <string name="desc_filesystems_message">معنی فایل‌سیستم مشخص است: مجموعه‌ای از کاربرها و یک ساختار پرونده است که اجازه فعالیت‌های شبه‌لینوکسی را می‌دهد.</string>
+
+    <!-- Supported Services -->
+    <string name="support_dropbear">SSH - Dropbear</string>
+    <string name="support_tightvnc">VNC - TightVNC</string>
+
+    <!-- Client Types -->
+    <string name="client_ssh">SSH</string>
+    <string name="client_vnc">VNC</string>
+
+    <!-- Alerts -->
+    <string name="button_ok">باشه!</string>
+    <string name="general_error_title">اتفاق پیش‌بینی نشده‌ای رخ داد!</string>
+    <string name="filesystem_credentials_reasoning">در حال ساخت یک فایل‌سیستم برای کاره شما هستیم. لطفا اطلاعات مرتبط خواسته‌شده را برای ما وارد کنید تا به عنوان پیش‌فرض این فایل‌سیستم استفاده شود.</string>
+
+    <!-- Play service error alerts -->
+    <string name="alert_play_service_error_message">برای تکمیل این کار، با مشکلی در دسترسی به خدمات گوگل پلی داریم. لطفا دوباره تلاش کنید و در صورت تداوم این مشکل، آن را با توسعه‌دهندگان در میان بگذارید.</string>
+    <string name="alert_play_store_required">کاره‌های پولی برای تایید پرداخت نیازمند دسترسی به خدمات گوگل‌پلی هستند. لطفا برای ادامه، آن را نصب کنید.</string>
+    <string name="alert_subscriptions_unsupported_message">به نظر می‌رسد نسخه گوگل پلی شما برای پشتیبانی اشتراک‌ها، بیش از حد قدیمی است. لطفا برای ادامه، آن را به‌روز کنید.</string>
+    <string name="alert_need_client_app_title">ناتوانی در یافتن کاره کارخواه!</string>
+    <string name="alert_need_client_app_message">برای اجام این کار به یک کارخواه SSH یا VNC نیزا دارد، اما به نظر می‌رسد پلی‌استور نصب نیست. لطفا برای ادامه، یک برنامه کارخواه SSH یا VNC نصب کنید.</string>
+
+    <!-- Permissions necessary alert -->
+    <string name="alert_permissions_necessary_title">یوزرلند نیاز به دسترسی دارد!</string>
+    <string name="alert_permissions_necessary_message">To continue, enable storage permissions to allow UserLAnd to download files and move them from external storage to app local storage.</string>
```

Comment:
Missing translation.

Reply:
I don't know why I have missed this line. I will make another commit now